### ISSUE TITLE
Clock not switching to 12 hr

### ISSUE THREAD

**Issue by jonathan-durbin** [gpakosz/.tmux#230]
I read through #44, but no solution was posted. How do I change the status bar clock to 12 hr format? 

I tried changing `tmux_conf_theme_clock_style='24'` to `tmux_conf_theme_clock_style='12'`, re-sourced my `.tmux.conf.local` file (e.g., <prefix> + r), but to no avail.

**Comment by gpakosz** [comment]
Hello @jonathan-durbin 👋,

It works for me.
After I changed `tmux_conf_theme_clock_style` from `24` to `12` in my `~/.tmux.conf.local` copy, I sourced the configuration and then I had to exit then re-enter clock mode with <kbd>CTRL</kbd>+<kbd>t</kbd>.

It it still doesn't work for you, you can verify the current clock mode by issuing
```
$ tmux show -wg clock-mode-style
```

**Comment by jonathan-durbin** [comment]
Thanks for the quick response, @gpakosz! I did what you suggested and I see a clock in the middle of the screen (I just have a single pane at the moment) in 12-hour format. I noticed, though, that the clock in the lower right part of the screen didn't change. Here's a screenshot:

![clock_24](https://user-images.githubusercontent.com/31511197/54643169-a19f3600-4a6c-11e9-8b8e-e8504a00be8a.png)

I just started becoming familiar with tmux and its capabilities - clock mode is very cool!
